### ISSUE TITLE
[fvt]2.12. xcatprobe xcatmn need modify output msg

### ISSUE THREAD

**Issue by junxiawang** [xcat2/xcat-core#1675]
env:all
build:[root@c910f02c01p14 subcmds]# lsxcatd  -v
Version 2.12.2 (git commit b7e752c2117f65c98916a0e3a93f45096765a605, built Wed Aug 10 06:16:31 EDT 2016)
[root@c910f02c01p14 subcmds]# 
How to reproduce:
```
[root@c910f02c01p14 subcmds]# xcatprobe xcatmn -i eth1

Dispatch cmd xcatmn to c910f02c01p08                                                                              [INFO]

-------------------------> I do not think this should before management's probe check information

NIC eth1 exists on current server                                                                                  [ OK ]
Get ip address of NIC eth1                                                                                        [ OK ]
The IP 10.2.1.14 of eth1 equals the value of 'master' in 'site' table                                             [ OK ]
IP 10.2.1.14 of eth1 is not a static ip on current server                                                         [WARN]
10.2.1.14 belongs to one of networks defined in 'networks' table                                                  [ OK ]
There is domain definition in 'site' table                                                                        [ OK ]
There is configuration in 'passwd' table for 'system' for node provision                          [ OK ]
There is /install directory on current server                                                                     [ OK ]
There is /tftpboot directory on current server                                                                  [ OK ]
The free space of / directory is more than 10 G                                                              [ OK ]
SELinux is disabled on current server                                                                             [ OK ]
Firewall is closed on current server                                                                              [ OK ]
HTTP service is ready on 10.2.1.14                                                                                [ OK ]
tftp tool isn't installed on current server, skip checking tftp service.                                          [WARN]
Please do probe again after installing tftp                                                                       
DNS server is ready on 10.2.1.14                                                                                  [ OK ]
The size of /var/lib/dhcpd/dhcpd.leases is less than 100M                                                         [ OK ]
DHCP service is ready on 10.2.1.14                                                                                [ OK ]
c910f02c01p08: bash: /opt/xcat/probe/subcmds/xcatmn: No such file or directory                                    [FAIL]

======================do ERROR summary=====================
[c910f02c01p08]:
        bash: /opt/xcat/probe/subcmds/xcatmn: No such file or directory
[root@c910f02c01p14 subcmds]# 

-------------------------->
 please modify error summary here and even no errors still show ======================do ERROR summary=====================
```

**Comment by junxiawang** [comment]
[root@c910f02c01p14 ~]# xcatprobe xcatmn -i eth0
[MN]: Sub process 'xcatd: SSL listener' is running                                                                [ OK ]
[MN]: Sub process 'xcatd: DB Access' is running                                                                   [ OK ]
[MN]: Sub process 'xcatd: UDP listener' is running                                                                [ OK ]
[MN]: Sub process 'xcatd: install monitor' is running                                                             [ OK ]
[MN]: Sub process 'xcatd: Discovery worker' is running                                                            [ OK ]
[MN]: Sub process 'xcatd: Command log writer' is running                                                          [ OK ]
[MN]: xcatd is listening on port 3001                                                                             [ OK ]
[MN]: xcatd is listening on port 3002                                                                             [ OK ]
[MN]: 'lsxcatd -a' works                                                                                          [ OK ]
[MN]: The value of 'master' in 'site' table is a IP address                                                       [ OK ]
[MN]: NIC eth0 exists on current server                                                                           [ OK ]
[MN]: Get ip address of NIC eth0                                                                                  [FAIL]
[MN]: Please use 'ip addr show' to check if there is ip assigned to eth0                                          
c910f02c01p08: bash: /opt/xcat/probe/subcmds/xcatmn: No such file or directory                                    [FAIL]

======================do summary=====================
[MN]: Check on MN FAILED.                                                                                         [FAIL]
        Get ip address of NIC eth0                                                                                [FAIL]
[SN:c910f02c01p08]: Check on SN c910f02c01p08 FAILED.                                                             [FAIL]
        bash: /opt/xcat/probe/subcmds/xcatmn: No such file or directory                                           [FAIL]
[root@c910f02c01p14 ~]# scp -r /opt/xcat/probe root@c910f02c01p08:/opt/xcat/
detect_dhcpd                                                                                          
------------------------> still errors about the output msg

**Comment by hu-weihua** [comment]
Fixed by pull request #1701, @junxiawang , could you help to verify this bug? thanks

**Comment by junxiawang** [comment]
verified
